internal/servicelog/formatter.go

```diff
@@ -30,7 +30,8 @@ type formatter struct {
 }
 
 const (
-	TimeFormat = time.RFC3339
+	// TimeFormat is RFC3339 with millisecond precision.
+	TimeFormat = "2006-01-02T15:04:05.999Z07:00"
```

Comment:
@niemeyer it was suggested by @benhoyt we might need slightly more precise timestamps for interleaving logs for multiple services. This would help when debugging multiple services. Regardless I think we can change this at any point in time because RFC3339 allows for any length fraction on the second.